api/src/main/java/org/openmrs/module/fhir2/api/search/interceptor/SummaryInterceptor.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
nit: This should actually be in the OMOD rather than the API, as it only comes into play when we're running the HAPI servlet. I would put it in the `omod/src/main/java/org/openmrs/module/fhir2/web/util` package.